test/unit/fetch-test.js

```diff
@@ -38,3 +38,44 @@ test('fetch without state', function (t) {
     t.equal(error.name, 'UnauthenticatedError', 'rejects with UnauthenticatedError error')
   })
 })
+
+test('fetch with non-string path', function (t) {
+  simple.mock(internals, 'fetchProperties').resolveWith({})
+
+  fetch({
+    url: 'http://example.com',
+    account: {
+      session: {
+        id: 'abc45678'
+      }
+    }
+  }, false)
+
+  t.deepEqual(internals.fetchProperties.lastCall.arg, {
+    url: 'http://example.com/session/account',
+    bearerToken: 'abc45678',
+    path: false
+  }, 'calls fetchProperties with account url and non-string path')
+  // expected behavior??
```

Comment:
>`false` makes not much sense here, why did you decide to test with it?

To achieve full branch coverage I need to have a test covering the `else` clause in the following code.

```javascript
if (typeof path === 'string') {
  set(state.account, path, properties)
} else {
  merge(state.account, properties)
}
```

Which, if I'm not mistaken, comes down to testing with a `path` where `typeof path !== 'string'`. I picked `false` *because* it doesn't make much sense, as I assumed that this was not a legitimate use case.

![coverage report excerpt](https://cloud.githubusercontent.com/assets/1736534/13351766/53f96c08-dc89-11e5-99c1-1818ca4e1d5c.png)